coffeecard/CoffeeCard/Services/AccountService.cs

```diff
@@ -48,57 +47,41 @@ public string Login(string username, string password, string version)
 
             ValidateVersion(version);
 
-            var user = _context.Users.FirstOrDefault(x => x.Email == username && x.IsVerified);
+            var user = _context.Users.FirstOrDefault(x => x.Email == username && x.IsVerified == true);
             if (user != null)
             {
                 var hashedPw = _hashService.Hash(password + user.Salt);
                 if (user.Password.Equals(hashedPw))
                 {
-                    var claims = new[]
-                    {
-                        new Claim(ClaimTypes.Email, username), new Claim(ClaimTypes.Name, user.Name),
-                        new Claim("UserId", user.Id.ToString())
-                    };
+                    var claims = new Claim[] { new Claim(ClaimTypes.Email, username), new Claim(ClaimTypes.Name, user.Name), new Claim("UserId", user.Id.ToString()) };
                     var token = _tokenService.GenerateToken(claims);
 
-                    // check for incomplete purchases
-                    _purchaseService.CheckIncompletePurchases(user);
+                    // check for incomplete purchases //TODO Fix this and reimplement
```

Comment:
Maybe we want to consider reimplementing this, or removing it altogether if we consider it redundant

Reply:
I'll think about it - what is the case or scenario where clipcards have not been delivered seen from an API perspective? Wouldnt the API not always deliver or if not able to implement a cancel reservation function?

Reply:
The only thing I can think of is if some unhandled error occurred while issuing the product, though I am not sure if it could/ should recover by itself in such an instance